README.md

```diff
@@ -278,6 +278,43 @@ manual folds from the syntax generated ones. More information about this
 problem and on how to configure the addon can be found on its GitHub
 page.
 
+## FAQ
+
+### How do enable/disable indenting in modules?
+There are two config variables for this. `b:verilog_indent_modules` (which will
+be eventually deprecated) and the newer `g:verilog_disable_indent`. Because of
+the existence of `b:verilog_indent_modules`, indenting in modules is disabled by
+default. As well as this, because `b:verilog_indent_modules` is a buffer
+vairable, to ensure indenting in modules is always enabled you must add
+the following to your .vimrc:
+```VimL
+    augroup systemverilog_settings_1
+        au!
+        au Filetype verilog_systemverilog let b:verilog_indent_modules = 1
+    augroup END
+```
+The same also applies for `b:verilog_indent_preproc` (indentation of
+preprocessor statements).
+
+### Why is opening verilog/systemverilog files so slow?
```

Comment:
I just tried `Konfekt/FastFold` and it made no difference and `LargeFile` disables syntax highlighting completely.

I did some experiments a while back and what I found was that adding `fold` to the `syn region` command just slowed things down a lot, regardless of whatever values the folding options where (`foldenable, foldmethod`, etc).

There was no configuration that involved having `syn region ... fold` and it not be slow. And given that, I dont think this is something a plugin can fix.